tests/unit/lib/document-test.js

```diff
@@ -126,7 +126,10 @@ module("Unit | Library | document", function(hooks) {
   test("computes the correct jexl context", async function(assert) {
     assert.expect(1);
 
-    assert.deepEqual(this.document.jexlContext, { form: "form" });
+    assert.deepEqual(this.document.jexlContext, {
+      form: "form",
+      info: { root: { form: "form" } }
```

Comment:
Would be nice if we have tests for `info.form` and `info.parent.form`

Reply:
I'll look into it! :smiley:

Reply:
I added a test for `info.form`. I omitted `info.parent.form` for now as the existing dummy data doesn't include subforms. I'm not sure on how to best extend the dummy data without impacting the other tests that rely on this structure.

(Tests are in unit/lib/field.)